multi-node.sh

```diff
@@ -64,18 +60,21 @@ ssh -T -o StrictHostKeyChecking=no $USER@$SERVER_IP <<-EOF && { echo "Build succ
   make -j 4
   make install
   echo "==> Building fabtests on second node"
-  cd $WORKSPACE
-  git clone https://github.com/ofiwg/fabtests
-  cd fabtests
-  git checkout $TARGET_BRANCH
+  cd $WORKSPACE/libfabric/fabtests
   ./autogen.sh
   ./configure --with-libfabric=$WORKSPACE/libfabric/install/ --prefix=$WORKSPACE/fabtests/install/ --enable-debug
   make -j 4
   make install
   # Runs all tests in the fabtests suite between two nodes while only expanding
   # failed cases
   echo "==> Running fabtests between two nodes"
-  LD_LIBRARY_PATH=$WORKSPACE/fabtests/install/lib/:$LD_LIBRARY_PATH BIN_PATH=$WORKSPACE/fabtests/install/bin/ FI_LOG_LEVEL=debug $WORKSPACE/fabtests/install/bin/runfabtests.sh -v $PROVIDER $CLIENT_IP $SERVER_IP
+  declare EXCLUDE=$WORKSPACE/fabtests/install/share/fabtests/test_configs/$PROVIDER/${PROVIDER}.exclude
```

Comment:
This wouldn't work work when we start running specific combinations of util providers (say PROVIDER=tcp;ofi_rxm and xyz;ofi_rxm), in which case we'll have to maintain a map of provider string to exclude file. This works for now though, I can refactor how we deal with exclude files when we expand the provider list in Jenkins.